### ISSUE TITLE
CreateGrant: people-parameter should be contact identifiers

### ISSUE THREAD

**Issue by andreas** [podio/podio-dotnet#14]
The [`CreateGrant`-method](https://github.com/podio/podio-dotnet/blob/0771066a5bccd4cfc6b85bdc756d16b4129bb80d/Podio%20.NET/Services/GrantService.cs#L103) currently accepts `List<Ref>` for the `people`-parameter. As mentioned in [the documentation](https://developers.podio.com/doc/grants/create-grant-16168841), this should be a list of [contact identifiers](https://developers.podio.com/doc/grants/create-grant-16168841), however. This means you cannot create grants for identifiers like these:

```
{"type": "mail",     "id": "example@example.com"}
{"type": "external", "id": {"linked_account_id": 123, "external_contact_id": "my_ext_id"}}
```

**Comment by agarcia00** [comment]
Thanks Andreas for opening this issue, this bug is affecting a development what I have been assisting on.

**Comment by AjmalVh** [comment]
Since the Id property can be assigned with different type of values like integer, string and object. I have changed the type of Id to dynamic.  The changes will available in Podio nuget package version 1.3.3
